src/main/java/programmers/org/voucher/constant/Command.java

```diff
@@ -1,19 +1,22 @@
 package programmers.org.voucher.constant;
 
-import programmers.org.voucher.exception.ErrorMessage;
-
 import java.util.Arrays;
 import java.util.NoSuchElementException;
 
+import static programmers.org.voucher.exception.ErrorMessage.INVALID_COMMAND;
+
 public enum Command {
     CREATE,
     LIST,
+    FIND,
+    UPDATE,
+    DELETE,
     EXIT;
```

Comment:
아래 형태가 데이터를 추가하고 삭제하는데 관리하기 편하실 거에요 :) 

```suggestion
    FIND,
    UPDATE,
    DELETE,
    EXIT
    ;
```

Reply:
36da880 커밋 피드백 반영
- 수정하였습니다 감사합니다!